### PR TITLE
[FIX] l10n_ar_purchase: Agregamos traducciones

### PR DESCRIPTION
**Pull request by mem-adhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina/pull/917).

File: l10n_ar_purchase/i18n/es.po [hunk 1]
```diff
@@ -83,3 +83,14 @@ msgstr "Pedido de compra"
 #: model_terms:ir.ui.view,arch_db:l10n_ar_purchase.report_purchasequotation_document
 msgid "Request for Quotation"
```
Comment (zaoral) [inline l10n_ar_purchase/i18n/es.po]: need to update manifest version to run update of the module and synchronize new translation terms?

File: l10n_ar_purchase/i18n/es.po [hunk 2]
```diff
@@ -83,3 +83,14 @@ msgstr "Pedido de compra"
...
+#. module: l10n_ar_purchase
+#: model_terms:ir.ui.view,arch_db:l10n_ar_purchase.custom_header_purchasequotation
+#: model_terms:ir.ui.view,arch_db:l10n_ar_purchase.custom_header_purchaseorder
+msgid "Date due:"
+msgstr "Fecha de vencimiento"
+
+#. module: l10n_ar_purchase
+#: model_terms:ir.ui.view,arch_db:l10n_ar_purchase.custom_header_purchaseorder
+msgid "Approve date:"
+msgstr "Fecha de aprobacion"
```
Comment (zaoral) [inline l10n_ar_purchase/i18n/es.po]: re order this terms where they should be.
To do this you can load this into a local database for example and then using Odoo tools export again the translations. apply for both files

File: l10n_ar_purchase/i18n/es.po [hunk 3]
```diff
@@ -83,3 +83,14 @@ msgstr "Pedido de compra"
 #: model_terms:ir.ui.view,arch_db:l10n_ar_purchase.report_purchasequotation_document
 msgid "Request for Quotation"
 msgstr "Solicitud de presupuesto"
+
+#. module: l10n_ar_purchase
+#: model_terms:ir.ui.view,arch_db:l10n_ar_purchase.custom_header_purchasequotation
+#: model_terms:ir.ui.view,arch_db:l10n_ar_purchase.custom_header_purchaseorder
+msgid "Date due:"
+msgstr "Fecha de vencimiento"
```
Comment (zaoral) [inline l10n_ar_purchase/i18n/es.po]: missing : on translation
```suggestion
msgstr "Fecha de vencimiento:"
```

File: l10n_ar_purchase/i18n/es.po (hunk 2 above)
Comment (zaoral) [inline l10n_ar_purchase/i18n/es.po]: same of above
```suggestion
msgstr "Fecha de aprobacion;"
```

**Review by zaoral — CHANGES_REQUESTED** [review]

**Comment by zaoral** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina#921

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina#921

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina#921

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina#921

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina#921

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina#921

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina#921

**Comment by fw-bot-adhoc** [comment]
@mem-adhoc @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina#921

**Pull request closed without merging**